build.sbt

```diff
@@ -70,7 +70,11 @@ ThisBuild / mimaBinaryIssueFilters ++= Seq(
   ),
   ProblemFilters.exclude[DirectMissingMethodProblem]("scodec.bits.ByteVector#HexDumpFormat.Ansi"),
   ProblemFilters.exclude[DirectMissingMethodProblem]("scodec.bits.ByteVector.toHexDumpNoAnsi"),
-  ProblemFilters.exclude[InaccessibleClassProblem]("scodec.bits.ByteVector$HexDumpFormat$Ansi$")
+  ProblemFilters.exclude[InaccessibleClassProblem]("scodec.bits.ByteVector$HexDumpFormat$Ansi$"),
+  ProblemFilters.exclude[MissingClassProblem]("scodec.bits.ByteVector$HexDumpFormat"),
+  ProblemFilters.exclude[MissingClassProblem]("scodec.bits.ByteVector$HexDumpFormat$"),
+  ProblemFilters.exclude[MissingClassProblem]("scodec.bits.ByteVector$HexDumpFormat$Ansi$"),
+  ProblemFilters.exclude[MissingFieldProblem]("scodec.bits.ByteVector.HexDumpFormat")
```

Comment:
Instead of adding exclusions (exclusions are forever) perhaps you can declare 1.1.32 cursed and not check bincompat against it. If we're lucky something like:
```scala
ThisBuild / tlMimaPreviousVersions ~= (_.filterNot(_ == "1.1.32"))
```